src/components/cards/wallet/WalletHint.tsx

```diff
@@ -0,0 +1,31 @@
+import { Wallet } from "@/types/wallet";
+import Currency from "@/components/ui/Currency";
+import Hint from "@/components/ui/Hint";
+import { useTranslation } from "next-i18next";
+import { ReactElement } from "react";
+
+interface WalletHintProps {
+  wallet: Wallet;
+}
+
+/**
+ * Wallet hint component
+ *
+ * @returns {ReactElement}
+ */
+
+export default function WalletHint({ wallet }: WalletHintProps): ReactElement {
+  const { t } = useTranslation();
+  return (
+    <>
```

Comment:
There is not need of empty fragment

Reply:
The map is returning multiple elements, we need these. fragments to return a single jsx